industrial_ci/mockups/legacy_catkin_lint_testpkg/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
This additional package is not needed..
There are other ways to check the version.